Sometimes generated bundles does not have main class in manifests

![Image](https://github.com/user-attachments/assets/a130b6b3-bd55-4061-b0f1-9c3386fc0027)

rusefi_autoupdate.jar manifest on the left side looks too scanty if compare it with normal manifest in uaefi bundle on the right side.

Bad `MANIFEST.MF`:
```
Manifest-Version: 1.0


```

Good `MANIFEST.MF`:
```
Manifest-Version: 1.0
Main-Class: com.rusefi.autoupdate.Autoupdate
Build-Jdk: 11.0.27 (Azul Systems, Inc. 11.0.27+6-LTS)
Built-Date: Tue May 20 11:55:54 UTC 2025


```

@kifir23917 it works!

https://github.com/rusefi/rusefi/actions/runs/15175449901/job/42674779771

```
  inflating: bundle_check/rusefi.snapshot.subaru_eg33_f7/console/STM32_Programmer_CLI/bin/libeay32.dll  
Archive:  bundle_check/rusefi.snapshot.subaru_eg33_f7/console/rusefi_console.jar
  inflating: manifest_check/META-INF/MANIFEST.MF  
Checking manifest in bundle_check/rusefi.snapshot.subaru_eg33_f7/console/rusefi_console.jar...
Archive:  bundle_check/rusefi.snapshot.subaru_eg33_f7/console/rusefi_autoupdate.jar
  inflating: manifest_check/META-INF/MANIFEST.MF  
Checking manifest in bundle_check/rusefi.snapshot.subaru_eg33_f7/console/rusefi_autoupdate.jar...
ERROR: 'META-INF/MANIFEST.MF' file in 'bundle_check/rusefi.snapshot.subaru_eg33_f7/console/rusefi_autoupdate.jar' doesn't contain 'Main-Class' definition
Archive:  bundle_check/rusefi.snapshot.subaru_eg33_f7/console/rusefi_ts_plugin_launcher.jar
  inflating: manifest_check/META-INF/MANIFEST.MF  
Checking manifest in bundle_check/rusefi.snapshot.subaru_eg33_f7/console/rusefi_ts_plugin_launcher.jar...
ERROR: some bundle .jar files have invalid minifest (for details see errors above)
Error: Process completed with exit code 1.
```

![Image](https://github.com/user-attachments/assets/9510d04f-c8ff-4994-affd-4c49789cefd6)

--- a/java_console/autoupdate/build.gradle
+++ b/java_console/autoupdate/build.gradle
@@ -21,7 +21,8 @@ evaluationDependsOn(':core_io')
 shadowJar {
   duplicatesStrategy = DuplicatesStrategy.FAIL
   base {
-    archivesName = "${rootDir}/../console/rusefi_autoupdate"
+    archiveFileName = 'rusefi_autoupdate.jar'
+    destinationDirectory = file("${rootDir}/../console")
     archiveClassifier = ''
   }
 

--- a/java_console/ui/build.gradle
+++ b/java_console/ui/build.gradle
@@ -27,7 +27,8 @@ dependencies {
 
 shadowJar {
     base {
-        archivesName = "${rootDir}/../console/rusefi_console"
+        archiveFileName = 'rusefi_console.jar'
+        destinationDirectory = file("${rootDir}/../console")
         archiveClassifier = ''
     }
 

